content/manuals/desktop/release-notes.md

```diff
@@ -29,6 +29,54 @@ For more frequently asked questions, see the [FAQs](/manuals/desktop/troubleshoo
 >
 > If you're experiencing malware detection issues on Mac, follow the steps documented in [docker/for-mac#7527](https://github.com/docker/for-mac/issues/7527).
 
+## 4.41.0
+
+{{< release-date date="2025-04-24" >}}
+
+{{< desktop-install-v2 all=true beta_win_arm=true version="4.41.0" build_path="/190394/" >}}
+
+### New
+
+- Docker Model Runner is now available on Windows machines with NVIDIA GPUs.
+- You can now push models to Docker Hub with Docker Model Runner.
+- Added support for Docker Model Runner's model management and chat interface in Docker Desktop for Mac and Windows (on hardware supporting Docker Model Runner). Users can now view, interact with, and manage local AI models through a new dedicated interface.
```

Comment:
@aevesdocker @kiview aren't this and the previous one about the same thing?

Reply:
2 is about CLI command, and 3 is about the new GUI feature, so LGTM.